semantic_conventions/http-common.yaml

```diff
@@ -4,11 +4,51 @@ groups:
     brief: "Describes HTTP attributes."
     prefix: http
     attributes:
-      - id: request.method
-        type: string
-        requirement_level: required
-        brief: 'HTTP request method.'
+      - id: request.canonical_method
+        type:
+          allow_custom_values: true
```

Comment:
Should this allow custom values though? If we want to offer this as a "control" for cardinality, I feel maybe we have to lock it down?